SampleViewer/SampleManager.cpp

```diff
@@ -408,8 +425,16 @@ void SampleManager::setDownloadProgress(double progress)
 
 void SampleManager::resetAuthenticationState()
 {
-  AuthenticationManager::credentialCache()->removeAndRevokeAllCredentials();
-  AuthenticationManager::setCredentialCacheEnabled(true);
+  // clear all credentials
+  ArcGISRuntimeEnvironment::authenticationManager()->arcGISCredentialStore()->removeAll();
+  auto f = ArcGISRuntimeEnvironment::authenticationManager()->networkCredentialStore()->removeAllAsync();
```

Comment:
what does this return? seems like we could do better for a variable name. Optional.